Source/WebKit/UIProcess/API/Cocoa/WKWebpagePreferences.mm

```diff
@@ -554,15 +554,34 @@ - (_WKWebsiteModalContainerObservationPolicy)_modalContainerObservationPolicy
     return WebKit::modalContainerObservationPolicy(_websitePolicies->modalContainerObservationPolicy());
 }
 
-#if USE(APPLE_INTERNAL_SDK)
-#import <WebKitAdditions/WKWebpagePreferencesAdditions.mm>
-#else
+- (BOOL)lockdownModeEnabled
+{
+    return _websitePolicies->captivePortalModeEnabled();
```

Comment:
Shouldn't this be protected with `#if ENABLE(LOCKDOWN_MODE_API)`?

Reply:
Hmm. I think you are right -- and return NO if not enabled.